### DISCUSSION TITLE
SSL certificate paths

### DISCUSSION THREAD

**Discussion by goldstar19** [felddy/foundryvtt-docker#333] [Q&A] [answered]
I have a docker container setup and when I add the two direct paths to SSL, it never seems to work.  I generated SSL certs from my domain and created the .crt and .key files but when I add the two environmental keys, it just doesn't work.  What might I be doing incorrectly below?  I am fairly new to docker and portainer (tried both portainer and docker-compose same issue) but some light direction would be helpful as I am trying to get the audio working in my container.

FOUNDRY_HOME | /home/foundry
-- | --
FOUNDRY_PASSWORD | secret
FOUNDRY_USERNAME | user
FOUNDRY_VERSION | 9.268
FOUNDRY_WORLD | gradient-descent
FOUNDRY_SSL_CERT | /mnt/data/foundry/cf.crt
FOUNDRY_SSL_KEY | /mnt/data/foundry/key.key
NODE_VERSION | 16.14.2
PATH | /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
YARN_VERSION | 1.22.18

Volumes:

Host/volume | /mnt/data/foundry
Path in container | /data

**Comment by felddy** [comment, marked as answer]
I think I see the issue.  Make sure you are passing paths that exist in the container's context, and not the host's.  

Try these values:

`FOUNDRY_SSL_CERT=/data/cf.crt`

`FOUNDRY_SSL_KEY=/data/key.key`

**Reply by goldstar19** [reply]
That was the trick, thank you!  You were right, I was thinking absolute path and not container path and once I did the switch, works like a charm.

**Comment by Kurokasa** [comment]
Hi, iam guessing i have the same issue but i still cant get it to work.
Iam using Docker Desktop for Windows (wich seems to make things more difficult here and there :D, but i have no other option right now)
I created a new Volume in Docker (LGVolume1) and put the files inside "LGVolume1/Data/SSL/key.key". If i understand ur answer up top correctly it should be working with:
docker run .... FOUNDRY_SSL_KEY=/Data/SSL/localhost.key ....
But it doesnt. I also tried saving it outside, but havent had much luck since it is inside the Windows PC then.
I would realy appreciate the help, i have never worked with docker befor and keeped banging my head for the last 2 days :D. Also, thx for the amazing work, it realy cool stuff like this exists.

**Reply by felddy** [reply]
The filesystem in the container is case sensitive.
Try setting `FOUNDRY_SSL_KEY` to `/data/SSL/localhost.key`.  Note the lowercased `data`.

**Comment by Kurokasa** [comment]
First of all thx for the super quick answer.
The Data Folder is capitalised (germans have a tendency to write stuff capitalised :D) i tried it anyway, and it didnt work.
Just to make it easy:
\\wsl.localhost\docker-desktop-data\data\docker\volumes\LGVolume1\Data\SSL\key.key
